docs/recipes/MigratingToMMKV.md

```diff
@@ -177,68 +178,68 @@ Now that you've moved the base storage functions over to MMKV, you might want to
 You may notice that the `storage.test.ts` test file will no longer pass. Replace the contents of this file with the following test data:
 
 ```tsx
-import { load, loadString, save, saveString, clear, remove } from "./storage"
-import { storage } from "./mmkv" // <- wherever your global `new MMKV()` constant is
+import { load, loadString, save, saveString, clear, remove } from "./storage";
```

Comment:
Are the addition of semicolons intended? No opinion either way, just wanted to check since it wasn't mentioned

Reply:
Probably an auto format thing, I'll double check with the consistency of the other snippets. Thanks!

Reply:
Going to leave it alone, seems as if the other recipes have it via the formatter.